modules/Core/pages/panel/pages.php

```diff
@@ -23,6 +23,48 @@
 // Load modules + template
 Module::loadPage($user, $pages, $cache, $smarty, [$navigation, $cc_nav, $staffcp_nav], $widgets, $template);
 
+function validate_page_form() {
+    global $language;
+    return Validate::check($_POST, [
+        'page_title' => [
+            Validate::REQUIRED => true,
+            Validate::MIN => 2,
+            Validate::MAX => 255
+        ],
+        'page_url' => [
+            Validate::REQUIRED => true,
+            Validate::MIN => 2,
+            Validate::MAX => 255,
+            Validate::NOT_START_WITH => '/panel'
```

Comment:
We need to consider subdirectory installations here

Reply:
Unfriendly urls don't even require to begin with / , so should make sure page path always begins with a /

Reply:
I believe the entered custom page path wouldn't contain subdirectories, so I don't think we need to do anything special to support subdirectory installations?

Reply:
yea you're right, we don't enter subdir there